### PR TITLE
blog(embertimes105): emberfest cfp reminder

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2019-07-05-the-ember-times-issue-105.md [hunk 1]
```diff
@@ -90,13 +91,13 @@ Ember Inspector running, so if you would be interested in helping out, please re
...
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [Reminder: EmberFest CFP Deadline August 1st 🎤](https://cfp.emberfest.eu/events/emberfest-2019)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+A quick reminder that the [EmberFest](https://emberfest.eu/) CFP deadline is fast approaching. We are just short of a month away from the August 1st deadline, so get your talk in today!
+
+Talks are 30 minutes long ranging from in-depth technical talks along with broader talks covering other aspects of software development.
```
Comment (ijlee2) [inline source/2019-07-05-the-ember-times-issue-105.md]:
```suggestion
Talks are 30 minutes long and can range from in-depth technical talks to broader talks covering other aspects of software development.
```

File: source/2019-07-05-the-ember-times-issue-105.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ help design the **Ember skill tree** 🌱,
 RFC to **support populating head tag** ⛑️,
 RFC to add `load` hook to Route 🎣
```
Comment (ijlee2) [inline source/2019-07-05-the-ember-times-issue-105.md]: Not related to your PR, Chris, but can we add commas to end of the lines 15 & 16 to separate items? Thanks.

**Review by ijlee2 — APPROVED** [review]
Looks great, @chrisrng, thank you!

File: source/2019-07-05-the-ember-times-issue-105.md (hunk 2 above)
Comment (chrisrng) [reply source/2019-07-05-the-ember-times-issue-105.md]: added!

**Pull request merged**